plugins/catalog-backend/src/database/operations/stitcher/performStitching.ts

```diff
@@ -180,6 +181,7 @@ export async function performStitching(options: {
     .map<EntityRelation>(row => ({
       type: row.relationType!,
       targetRef: row.relationTarget!,
+      metadata: row.metadata ? JSON.parse(row.metadata) : undefined,
```

Comment:
`null` handling here and above is a bit weird because of this. I don't think that's an issue, but was trying to work around `JSON.parse(undefined)` throwing.